### ISSUE TITLE
versions:display-dependency-updates goal need to have excludes and includes property

### ISSUE THREAD

**Issue by JithuPaul24** [mojohaus/versions-maven-plugin#318]
At present, if the following command is executed:
**mvn versions:display-dependency-updates**
```   <plugin>		
    <groupId>org.codehaus.mojo</groupId>
    <artifactId>versions-maven-plugin</artifactId>
    <inherited>false</inherited>
    <configuration>
    <allowSnapshots>false</allowSnapshots>
    <outputFile>build/outdated-dependencies.txt</outputFile>
    <rulesUri>file:///C:rules.xml</rulesUri>
    <processDependencyManagement>false</processDependencyManagement> 
    <processPluginDependencies>false</processPluginDependencies>  
    </configuration>
    </plugin>
```
**The plugin currently scans for latest updates for all the dependencies listed in the pom file.**
In certain cases, we might need to scan the updates to a limited dependency or a particular classifier or scope.
The display-dependency-updates doesn't currently allow this to be done.
But, other goals like versions:use-latest-versions allows me to have the excludes and includes property in them.

**Comment by markkolich** [comment]
+1 this would be tremendously helpful in cases where a stack is pegged at a specific version of an artifact (as required by other upstream dependencies), and can't be upgraded until all upstream dependencies upgrade in parallel.

E.g., `javax.servlet:javax.servlet-api` &mdash; my application stack can't migrate forward to version 4.x of the Servlet API until Jetty does.

**Comment by HonoluluHenk** [comment]
Implemented PR #350 with a possible fix

**Comment by ymartin59** [comment]
Yes, I am interested in scanning only compile and test scopes, ignoring provided ones.
